usr/lib/dbus-property-service/propertyservice

```diff
@@ -22,22 +22,20 @@ class PropHelper():
         prop = str(pipe, 'ascii').rstrip()
         return prop
 
-    def GetBoolean(property):
+    def GetServiceStatus(service_name):
         retval = 0
-        cmd = "getprop"
-        pipe = subprocess.check_output([cmd, property])
-        if str(pipe, 'ascii').rstrip() == 'true':
+        cmd = "/sbin/status"
+        pipe = subprocess.check_output([cmd, service_name])
+        if 'running' in str(pipe, 'ascii'):
             retval = 1
         return retval
```

Comment:
I wonder if it's better to check the process's return code...

Also, I guess we don't have to define `cmd` separately. Having it inline might helps mentally following the code easier.

Reply:
> I wonder if it's better to check the process's return code...

`check_output` already does that, and throws an exception is the return code is not 0. As a matter of fact, the `status` command returns 0 if the command itself was successful, regardless of the status of the service. That's why we are checking the stdout.

> Also, I guess we don't have to define cmd separately. Having it inline might helps mentally following the code easier.

You have a point there. I changed it now.